src/components/project-list.jsx

```diff
@@ -1,21 +1,43 @@
+import { graphql, Link, useStaticQuery } from 'gatsby'
+import Img from 'gatsby-image'
 import React from 'react'
 import styled from 'styled-components'
 import media from 'styled-media-query'
-import { Link } from 'gatsby'
-import PageContainer from './page-container'
 import background from '../assets/howly-bg.svg'
-import logo from '../assets/images/howly-logo.png'
+import PageContainer from './page-container'
 
-const Project = props => (
-  <ProjectContainer to="/howly">
-    <img src={logo} alt="Howly logo" />
-    <h2>
-      React Native, Node.js, AWS,
-      <br /> Serverless, Product Design
-    </h2>
-    <p>The snapchat of events, Howly keeps you and your friends connected</p>
-  </ProjectContainer>
-)
+const Project = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      howlyLogoImg: file(relativePath: { eq: "howly-logo.png" }) {
+        childImageSharp {
+          fluid(maxHeight: 1000) {
+            ...GatsbyImageSharpFluid_tracedSVG
```

Comment:
I’ve been using blurred vs traced for the rest of the site

Reply:
No worries, I'll change it, thought it looked better using traced though! As it outlined the word Howly.